src/components/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,389 @@
+    },
+  },
+  {
+    type: "action",
+    id: "item-title-2",
+    label: "I'm item 2 and I'm also very very long",
+    onClick: () => {
+      console.log("Item Title 2 clicked");
+    },
+  },
+  {
+    type: "action",
+    id: "item-title-3",
+    label: "I'm item 3",
+    onClick: () => {
+      console.log("Item Title 3 clicked");
+    },
+  },
+];
+
+/**
+ * Main Story for the Menu component. This must contain the `args`
+ * and `parameters` properties in this object.
+ */
+export const WithControls: Story = {
+  args: {
+    className: undefined,
+    id: "Menu-id",
+    labelText: "I'm the menu",
+    showLabel: true,
+    showBorder: true,
+    listAlignment: "left",
+    highlightColor: "blogs",
+    selectedItem: undefined,
+    listItemsData: undefined,
+  },
+  render: (args) => (
+    <Menu
+      {...args}
+      labelText={args.labelText}
+      listItemsData={defaultListItems}
+    />
+  ),
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=80460%3A101982",
+    },
+    jest: ["Menu.test.tsx"],
+  },
+};
+
+// The following are additional Menu Stories.
+
+export const MenuTypes: Story = {
+  render: () => (
+    <HStack>
+      <Menu labelText={"Default Button"} listItemsData={defaultListItems} />
+      <Menu
+        showBorder={false}
+        labelText={"Text Button"}
+        listItemsData={defaultListItems}
+      />
+      <Menu
+        showLabel={false}
+        labelText={"Menu"}
+        listItemsData={defaultListItems}
+      />
+      <Menu
+        showBorder={false}
+        labelText={"Menu"}
+        showLabel={false}
+        listItemsData={defaultListItems}
+      />
+    </HStack>
+  ),
+};
+
+export const MenuContent: Story = {
+  render: () => (
+    <HStack>
+      <Menu labelText={"I have images"} listItemsData={imageListItems} />
+      <Menu labelText={"I have icons"} listItemsData={iconListItems} />
+      <Menu labelText={"I have groups"} listItemsData={longListItems} />
+      <Menu
+        labelText={"I have max height"}
+        listItemsData={reallyLongListItems}
+      />
+      <Menu labelText={"I have max width"} listItemsData={wideListItems} />
+    </HStack>
+  ),
+};
+
+export const MenuHighlightColors: Story = {
+  render: () => (
+    <HStack>
+      <Menu
+        labelText={"Blogs (default) "}
+        highlightColor={"blogs"}
+        selectedItem={"item-title-1"}
+        listItemsData={defaultListItems}
+      />
+      <Menu
+        labelText={"Research"}
+        highlightColor={"research"}
+        selectedItem={"item-title-1"}
+        listItemsData={defaultListItems}
+      />
+      <Menu
+        labelText={"LPA"}
+        highlightColor={"researchLibraryLpa"}
+        selectedItem={"item-title-1"}
+        listItemsData={defaultListItems}
+      />
+      <Menu
+        labelText={"Education"}
+        highlightColor={"education"}
+        selectedItem={"item-title-1"}
```

Comment:
No need to change this but just pointing out that `{"string-value"}` is the same as `"string-value"` for react props.